### ISSUE TITLE
Can't create account on Nextcloud Notes/Deck

### ISSUE THREAD

**Issue by skarmoutsosv** [nextcloud/Android-SingleSignOn#202]
When I select to add an account from the menu of Nextcloud Notes or Nextcloud Deck, the application shows at the center of the screen "Select account" and nothing more. If I press anywhere outside the message, the message disappears and a moving circle is shown for ever.
In contrary Nextcloud and Nextcloud Talk both work as expected.

An issue created initially at https://github.com/stefan-niedermann/nextcloud-notes/issues/784 (includes a screenshot), and followed by the current issue here as suggested.

My versions:
Nextcloud: 3.11.1
Nextcloud Talk: 8.0.9
Nextcloud Notes: 2.11.12
Nextcloud Deck: 1.0.1
DOOGEE-Shoot_1-Android6.0-20170118

**Comment by tobiasKaminsky** [comment]
Hm. You should see at least an empty list, and a "add account" button.

**Comment by stefan-niedermann** [comment]
There's a screenshot in the linked issue, no empty list and no "add account" visible...

**Comment by tobiasKaminsky** [comment]
Hm, for me it is working, at least I do see the list. 

Can you provide us additional infos via logcat? https://github.com/nextcloud/android/blob/master/README.md#getting-debug-info-via-logcat

**Comment by skarmoutsosv** [comment]
No luck with logcat.

Logcat using a PC:
My computer misbehaves when trying to connect using a USB cable (mouse freezes).

Logcat using my phone:
I can't find a way to root my phone, so, Catlog does not produce something usefull.

Do you have any suggestions?

**Comment by tobiasKaminsky** [comment]
Is this still valid?

**Comment by David-Development** [comment]
@tobiasKaminsky Maybe this is indeed related to https://github.com/nextcloud/Android-SingleSignOn/issues/68

And the fact that the spinner keeps on spinning after he clicked outside the dialog is probably caused since the callback is not called..?

**Comment by skarmoutsosv** [comment]
Finally I managed to use logcat using the following command
`adb logcat | grep "$(adb shell ps | awk '/com.nextcloud.client/{print $2}')" > logcatOutput.txt`
[logcatOutput.txt](https://github.com/nextcloud/Android-SingleSignOn/files/4947439/logcatOutput.txt)

While logcat was running I opened NextcloudDeck, pressed select account and the message "Select an account" appeared, without any spinner (moving circle) or any other text. I pressed outside this message and returned to the previous screen. Then exit NextcloudDeck and Ctrl+C logcat. The created logcat file is attached.

My NextcloudDeck version is 1.8.2.

**Comment by tobiasKaminsky** [comment]
But you do have accounts set up in your NC Files app?

**Comment by skarmoutsosv** [comment]
Yes. I have two accounts, they work fine and I can manage all the files through nextcloud files app. My server is at http://cloud.skarmoutsosv.gr

**Comment by skarmoutsosv** [comment]
The issue appeared while using a Doogee Shoot1 (Android 6).
Everything works fine using a new phone (Android 11).

**Comment by christianlupus** [comment]
I am having a Cubot Max (I think) and it is running Android 6 as well. Same problem.
The phone is the secondary phone mainly lying in the apartement and serving as a digital sheet of paper (notes app). Thus, it is not the most up-to-date android version.
